### ISSUE TITLE
The core howto index should link to the endpoints howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4773]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4773|
|Type|enhancement|
|Created|2010-12-14 18:32:15Z|

http://twistedmatrix.com/documents/current/core/howto/ should link to http://twistedmatrix.com/documents/current/core/howto/endpoints.html

Attachments:

* [fix-howto-doc-list.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/557/5577fb3152eb7d43e69669eba5da25e0ddde59eb/ba5de6afd5173d971cdce147203792c53a8ee1e5.diff) (465 bytes) - added by cyli on 2011-01-16 20:28:51Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4773 4773
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1292351535000000 1292351535000000
changetime__1295215445000000 1295215445000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_1"></a>|@cyli removed owner|
|-|-|

Stuck it in the section titled "Tutorials" under the "Writing a TCP Server" and "Writing a TCP Client" links, since both of them refer to endpoints and are referred to by the endpoints tutorial.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @cyli|
|-|-|

Please merge with a .doc newsfile, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli set status to `closed`|
|-|-|

(In [30476]) Apply patch fix-howto-doc-list.diff: Link to endpoints tutorial

Author: cyli
Reviewer: thijs
Fixes: [#4773](https://github.com/twisted/twisted/issues/4773)

Link to endpoints tutorial from main Twisted core howtos list.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
